scripts/doc-generation/.jazzy.yaml

```diff
@@ -0,0 +1,11 @@
+module: MapboxMaps
+title: Mapbox Maps SDK for iOS
+author: Mapbox
+theme: jazzy-theme
+sdk: iphonesimulator
+swift_build_tool: xcodebuild
+build_tool_arguments:
+  - -scheme
+  - MapboxMaps
+  - -destination
+  - generic/platform=iOS Simulator
```

Comment:
yamllint requests that this file end with a newline

Reply:
Looks like I also have to finish file with `...` syntax.